### ISSUE TITLE
approach to documenting & sharing models

### ISSUE THREAD

**Issue by kmcdono2** [Living-with-machines/MapReader#293]
Options:
- [ ] huggingface only
- [ ] scivision catalog, thus model repo [template from scivision - https://scivision.readthedocs.io/en/latest/model_repository_template.html#model-repo-template]
- [ ] model repo + .....
- [ ] ?

**Comment by kmcdono2** [comment]
@rwood-97 - I noticed the models you loaded to HuggingFace are under your profile (np) - let's figure out how to get them added to the LwM org list of models (they don't appear there now)

**Comment by rwood-97** [comment]
Yep, I can do this asap, will double check they are all the most recent checkpoints and then move to the LwM org.

**Comment by kallewesterling** [comment]
Did we ever made a decision about this, @rwood-97? Happy to add something in the documentation about it, if we decided to go with HuggingFace?
